src/frontend/dashboard-layout.js

```diff
@@ -88,7 +88,7 @@ _.extend(DashboardLayout.prototype, {
 				else {
 					$layout.addClass('knsh-dashboard-layout-group__m-col');
 				}
-		
+				
```

Comment:
Ah, no big deal, I was more worried that it looked like spaces got replaced by tabs... but you use tabs anyway, so I shouldn't really worry about it :)